deployment/README.md

```diff
@@ -121,8 +121,8 @@ the job at 12 A.M. (00:00) daily.
 
 ### Configurable variables in template
 
-- AWS_BUCKET_NAME_MODEL - This is the bucket from which the models will be read.
-- AWS_BUCKET_NAME_INFERENCE - This is the bucket to which the inference results will be written
+- S3_MODEL_BUCKET - This is the bucket from which the models will be read.
```

Comment:
Arun we should update [Initial Setup of secrets](https://github.com/kubesecurity/openshift-probable-vulnerabilities/tree/master/deployment#initial-setup-of-secrets) section in readme file as well, as currently old information is present there.

Reply:
Thanks @rajusem. Just updated.